_posts/on-non-verbal-communication.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+header_image: /img/posts/on-non-verbal-communication/header.jpg
+title: "On non-verbal communication for remote workers"
+author: Mauro Servienti
+synopsis: "Communication is hard. Bad communication leads to misunderstandings, stress, and bad things. Remote communication is even harder; what could go wrong? Can we do something about it?"
+tags:
+- remote working
+- remote teams
+- communication
+---
+
+We constantly communicate, even if we're not talking. Communication comes in many flavors. There is verbal and non-verbal communication. Verbal communication is pretty straightforward: we say words—maybe words meaning nothing. Still, verbal communication remains :-)
+
+Non-verbal communication, as the name implies, is the one that doesn't use words coming from our mouths. There are two types of non-verbal communication: There is the psychological side of non-verbal communication, such as, for example, the postures we assume when in a conversation, and, on the other side, all that we communicate using gestures and gesticulating. And trust me, gesticulating, we can share a lot. I'm Italian. We master gesticulations.
+
+Let me go straight to the problem. Remote workers can only rely on words. We can only use verbal communication. Postures don't go through during video calls, and the same happens for gesticulating. Usually, the problem is that the camera view angle is too narrow to make so that our body and sometimes even ours hands are visible. And no, increasing the camera view angle doesn't solve any problem. It makes it worse. People on the call will be perceived distant as if they were sitting at an incredibly large table.
```

Comment:
```suggestion
Let me go straight to the problem. Remote workers can only rely on words. We can only use verbal communication. Postures don't go through during video calls, and the same happens for gesticulating. Usually, the problem is that the camera view angle is too narrow to make so that our body and sometimes even ours hands are visible. And no, increasing the camera view angle doesn't solve the problem. In fact it can make it worse because people on the call will be perceived as distant, as if they were sitting at an incredibly large table.
```